general/project/projectinit.go

```diff
@@ -147,8 +146,8 @@ func (pic *ProjectInitCommand) createBuildMessage(technologiesMap map[coreutils.
 			"\n"
 	}
 	if ok := technologiesMap[coreutils.Docker]; ok {
-		baseurl := strings.TrimLeft(pic.serverUrl, "https://")
-		baseurl = strings.TrimLeft(baseurl, "http://")
+		baseurl := strings.TrimSuffix(strings.TrimSpace(pic.serverUrl), "https://")
+		baseurl = strings.TrimSuffix(baseurl, "http://")
```

Comment:
Shouldn't trim prefix?
```suggestion
		baseurl := strings.TrimPrefix(strings.TrimSpace(pic.serverUrl), "https://")
		baseurl = strings.TrimPrefix(baseurl, "http://")
```

Reply:
Good catch! thanks